anaconda.py

```diff
@@ -300,8 +224,9 @@ def setup_environment():
         sys.exit(0)
 
     log.info("%s %s", sys.argv[0], util.get_anaconda_version_string(build_time_version=True))
-    if os.path.exists("/tmp/updates"):
-        log.info("Using updates in /tmp/updates/ from %s", opts.updateSrc)
+
+    if opts.updates_url:
+        log.info("Using updates from: %s", opts.updates_url)
```

Comment:
Aren't we missing the log message in case of updates defined in kickstart ?
(And if so, do we care?)

Reply:
I guess we can log both. I will fix that in a follow-up PR.